demo/src/components/ChannelController.js

```diff
@@ -41,6 +41,9 @@ const getPixelValueDisplay = (isOn, pixelValue, shouldShowPixelValue) => {
   if (typeof pixelValue === 'number') {
     return <Typography> {pixelValue} </Typography>;
   }
+  if (pixelValue === FILL_PIXEL_VALUE) {
+    return <Typography> {FILL_PIXEL_VALUE} </Typography>;
+  }
```

Comment:
What else could pixel value be? Is there a reason we can't just have:

```javascript
if (pixelValue) {
      return <Typography> {pixelValue} </Typography>;
}
```

Reply:
Good point!